packages/server/test/course.integration.ts

```diff
@@ -323,32 +325,34 @@ describe('Course Integration', () => {
     });
   });
 
-  it('tests deleting override using the endpoint', async () => {
-    const course = await CourseFactory.create();
-    const user = await UserFactory.create();
-    const professor = await UserFactory.create();
-    await UserCourseFactory.create({
-      user: professor,
-      role: Role.PROFESSOR,
-      course,
-    });
-    await UserCourseFactory.create({
-      user: user,
-      role: Role.TA,
-      override: true,
-      course,
-    });
+  describe('POST /courses/:id/update_override', () => {
```

Comment:
I think you want this to be a delete

```suggestion
  describe('DELETE /courses/:id/update_override', () => {
```